test/cache.test.ts

```diff
@@ -0,0 +1,28 @@
+import { Cache } from "../src/cache/page_desc_cache.ts";
+import { GitbookSpaceClient } from "../src/gitbook_client.ts";
+import { spy, Spy, stub } from "https://deno.land/x/mock@v0.9.5/stub.ts";
+import { mock1 } from "./__mocks__/gitbook_content_mock.js";
+import { assert } from "https://deno.land/std@0.97.0/testing/asserts.ts";
+import { FakeTime } from "https://deno.land/x/mock@v0.9.5/time.ts";
```

Comment:
Great! I was worried we'd have to actually wait long time period, and eat up Action free minutes 🎉